### PR TITLE
Use commonmark 2.0.0 which is what Laravel 8 already uses

### PR DESCRIPTION
**Pull request by susanBuck**
Reference: https://github.com/GrahamCampbell/Laravel-Markdown/issues/158

### CODE REVIEW

**Comment by bastinald** [comment]
might be better to do `^1.5 || ^2.0` no?

**Comment by victor-priceputu** [comment]
Can this or #160 be merged in, please? :)

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "illuminate/filesystem": "^6.0 || ^7.0 || ^8.0",
         "illuminate/support": "^6.0 || ^7.0 || ^8.0",
         "illuminate/view": "^6.0 || ^7.0 || ^8.0",
-        "league/commonmark": "^1.5"
+        "league/commonmark": "^2.0.0"
```
Comment (yannoff) [inline composer.json:18]: To avoid BC breaks, it would be safer to allow either versions : 

```suggestion
        "league/commonmark": "^1.5 || ^2.0"
```

Comment (mcadet) [reply composer.json:18]: oh @yannoff using Laravel ! Insane!

Comment (yannoff) [reply composer.json:18]: > oh @yannoff using Laravel ! Insane!

@mcadet I just sense it's time to leave symfony before the wreck :stuck_out_tongue_closed_eyes:

**Comment by GrahamCampbell** [comment]
Thanks for the PR. This package can still be installed on Laravel 8, without this change. Add the package by hand to your `composer.json` file and run `composer update`. I'll be releasing a new major version of this package that switches from commonmark v1 to v2 in the near future.

**Pull request closed without merging**